styles.css

```diff
@@ -1,16 +1,45 @@
-body {
+body, html {
   font-family: "Montserrat", sans-serif;
+  height: 100%;
+  width: 100%;
+}
+
+input, label {
+  margin:10px;
+}
+
+#books {
```

Comment:
Even though you're using id selectors in the javascript, lets use class selectors only in the CSS. Also, group the css by section with a comment header labeling each section like: `/* LAYOUT */`  `/* BOOKS */` `/* FORM */` etc.